src/settings_type.h

```diff
@@ -117,6 +117,7 @@ struct GUISettings {
 	uint8  window_soft_limit;                ///< soft limit of maximum number of non-stickied non-vital windows (0 = no limit)
 	ZoomLevel zoom_min;                      ///< minimum zoom out level
 	ZoomLevel zoom_max;                      ///< maximum zoom out level
+	ZoomLevel sprite_zoom_min;               ///< maximum zoom level at which higher-resolution alternative sprites will be used (if available) instead of scaling a lower resolution sprite
```

Comment:
suggestion: `Highest resolution sprites (if available) to use before scaling lower resolution sprites`?

Possibly also something similar for the commit message? I really like the wording you used for the setting, at least to me, is very clear.

Reply:
I don't like my own suggestion .. ugh, I now know what you mean, but I also see it is difficult to describe in a single sentence :P What-ever what you have now is fine :)